lib/product-import.coffee

```diff
@@ -212,17 +219,25 @@ class ProductImport
     skuString = "sku in (#{skus.map((val) -> JSON.stringify(val))})"
     "masterVariant(#{skuString}) or variants(#{skuString})"
 
+  _doesProductHaveSku: (product) ->
+    if product.masterVariant?.sku
+      return true
+
+    if product.variants?.length
+      for variant in product.variants
+        if variant.sku
+          return true
+    false
```

Comment:
Is it enough that at least 1 variant has SKU only? How sync will behave if only masterVariant has SKU and variants not - could variants without SKUs be matched/updated later without issues?